translations/combat.tex/he.po_bkp

```diff
@@ -0,0 +1,634 @@
+# Hebrew translations for Homm package
```

Comment:
What is this file?

Reply:
backup I don't need anymore... deleting...